tests/08-native-runs/15-ieee802154-security/Makefile

```diff
@@ -0,0 +1,9 @@
+CONTIKI_PROJECT = test-ieee802154-security
+all: $(CONTIKI_PROJECT)
+
+TARGET = native
+
+MODULES += os/services/unit-test
+
+CONTIKI = ../../..
```

Comment:
I realize the PR has been in the queue for a while, but there is no need to define CONTIKI nowadays.